docs/getting-started/edfi-exchange/how-it-works.md

```diff
@@ -0,0 +1,27 @@
+---
+labels:
+ - exchange-content
+---
+
+# How it Works
+
+## Overview
+
+The **Ed-Fi Exchange** is a technology hub for community contributions aligned to the Ed-Fi Data Standard and Implementation Suite.
+
+![Ed-Fi Exchange Graphic](https://edfidocs.blob.core.windows.net/$web/img/edfi-exchange/exchange_graphic.png)
+
+## How the Ed-Fi Exchange Benefits the Community
+
+**States**, **Districts**, **Technology Providers** and the **Ed-Fi Alliance** contribute solutions to the Ed-Fi Exchange that may be downloaded by other Ed-Fi Community members. Ed-Fi Exchange contributions may take the form of extensions to the Ed-Fi Data Standard, the Ed-Fi ODS and API, and Ed-Fi Dashboards in the form of plugins. In addition, conceptual documents such as technical white papers and implementation project overviews may be contributed to assist other licensees planning to leverage Ed-Fi technology.
+
+The goal is to make the Ed-Fi Community **greater than the sum of its parts** by openly sharing innovations aligned to the Ed-Fi Data Standard.
+
+## How to Fully Access Contributions
+
+Only Ed-Fi Licensees can **download contributions** and **view contributing organization information** to learn more about their Ed-Fi aligned solutions. To get full access to the Ed-Fi Exchange, you can [request a license](#).
+
+Once the Ed-Fi Alliance has verified your status as a licensee, you will receive an e-mail with instructions to log into the site. You will then be able to view additional solution detail, download contributions and contact solution contributors.
```

Comment:
How embarrassing that we didn't clean this up in Confluence! Please delete lines 20-24 as no longer relevant. Since 2020 🤕.